src/Microsoft.ML.Torch/TorchModel.cs

```diff
@@ -32,14 +33,32 @@ internal TorchModel(IHostEnvironment env, TorchSharp.JIT.Module module, string m
             ModelPath = modelLocation;
         }
 
+        // REVIEW: we need to figure out what this was doing in tensorflow models and whether it makes sense to have in torch models.
+        // I guess it's a utility method to inspect the schema of the model when you load the model. Is it the input schema or the output schema?
+        public DataViewSchema GetModelSchema()
+        {
+            throw new NotImplementedException();
+        }
+
         /// <summary>
-        /// Get <see cref="DataViewSchema"/> for complete model.
-        /// Every node in the Torch model will be included in the <see cref="DataViewSchema"/> object.
+        /// Scores a dataset using a pre-traiend <a href="https://www.pytorch.org/">Torch</a> model.
```

Comment:
I think we should say that this is for TorchScript models only.

Reply:
I missed this, sorry! But I think we definitely need to update the documentation.